content/en/docs/concepts/architecture/nodes.md

```diff
@@ -160,6 +160,16 @@ to be unreachable. (The default timeouts are 40s to start reporting
 ConditionUnknown and 5m after that to start evicting pods.) The node controller
 checks the state of each node every `--node-monitor-period` seconds.
 
+In versions of Kubernetes prior to 1.13, NodeStatus is the heartbeat from the
+node. Starting from Kubernetes 1.13, node lease feature is introduced, so that
```

Comment:
Can you add a bit more information about what the node lease is? Perhaps linking to the KEP might be helpful too.

Reply:
Oh can you also add what state (alpha/beta/etc) the node lease feature is in? Thanks!